### ISSUE TITLE
Inconsistent "Image Fit" behaviour

### ISSUE THREAD

**Issue by d4niel-P** [noinnion/greader#401]
"Image Fit" switch resets on every new page to "Normal img width".

Here's short clip showing the issue:
[![gReader Img Fit](http://img.youtube.com/vi/aSR45sI9YLk/0.jpg)](http://www.youtube.com/watch?v=aSR45sI9YLk "gReader Img Fit")

**Comment by noinnion** [comment]
what is your feed settings? long press on a feed and select Preferences, is Image fit enabled?

**Comment by d4niel-P** [comment]
Yes, it's enabled. I've tried every l combination in settings and still I have to switch it on every page.

![Screenshot_20210225-223829_gReader](https://user-images.githubusercontent.com/4502783/109223088-737b1280-77ba-11eb-9cdc-4a02aa679fc9.jpg)
